pallets/subtensor/src/math.rs

```diff
@@ -1223,3 +1223,21 @@ pub fn mat_ema_alpha_vec(
     }
     result
 }
+
+/// Return the quantile of a vector of I32F32 values.
+pub fn quantile(data: &Vec<I32F32>, quantile: f64) -> I32F32 {
+    let mut sorted_data = data.clone();
+    sorted_data.sort_by(|a, b| a.partial_cmp(b).unwrap());
```

Comment:
Can we do `cmp` instead of `partial_cmp` here? The `unwrap` can panic.

Reply:
done